common.sh

```diff
@@ -430,26 +433,38 @@ if [[ ! -z ${AGENT_E2E_TEST_SCENARIO} ]]; then
           export NUM_MASTERS=5
           export MASTER_VCPU=4
           export MASTER_DISK=100
+          if [[ "$AGENT_OPERATORS" =~ "mtv" ]]; then
+             export MASTER_VCPU=9
```

Comment:
As `MASTER_VCPU` is always set to `9` irrespective of the cluster topology, it could be moved out of the switch case and overrriden only once at one place.

Reply:
Good point. Fixed.